setup.py

```diff
@@ -33,5 +49,14 @@
         'toml',
         'python-dotenv',
         'twine',
+        'boto3',
     ],
+    license='Apache 2.0',
+    long_description_content_type='text/markdown',
+    long_description=long_description,
+    name=PACKAGE_NAME,
+    packages=find_packages(),
```

Comment:
Any reason for using `find_packages()` vs `packages=['mbed_tools_ci_scripts']`? 🤔

Reply:
this is how you set it up

Reply:
D'oh. Let me answer my own question then: "No reason and you should be ashamed of yourself for lack of consistency".

Reply:
these were not my words ;)

Reply:
lovin' the self-punishment there...